_code-samples/key-derivation/js/index.js

```diff
@@ -112,27 +145,25 @@ class Seed {
         // Almost everyone just uses the first account, `0`.
         const accountIndex = opts.accountIndex || 0;
 
-        return this._deriveScalar(publicGen.encodeCompressed(), accountIndex)
-            .add(privateGen)
-            .mod(order);
+        const scalar = this._deriveScalar(publicGen.encodeCompressed(), accountIndex)
+        return BigNumber(scalar).plus(privateGen).mod(order);
     }
 
     _deriveScalar(bytes, discrim) {
-        const order = Secp256k1.curve.n;
-        for (let i = 0; i <= 0xffffffff; i++) {
-            // We hash the bytes to find a 256 bit number, looping until we are sure it
+        const order = Secp256k1.curve.n
```

Comment:
minor nit: if possible, I'd prefer to see the semi-colons at the end of the lines in this file